src/main/java/shadows/apotheosis/adventure/affix/socket/SocketHelper.java

```diff
@@ -54,9 +56,17 @@ public static void setGems(ItemStack stack, List<ItemStack> gems) {
 	}
 
 	public static int getSockets(ItemStack stack) {
-		var inst = AffixHelper.getAffixes(stack).get(Affixes.SOCKET.get());
-		if (inst == null) return 0;
-		return (int) inst.level();
+		AffixInstance socketAffix = AffixHelper.getAffixes(stack).get(Affixes.SOCKET.get());
+		int sockets = 0;
+		if (socketAffix != null) {
+			sockets = (int) socketAffix.level();
+		}
```

Comment:
Can be simplified as a ternary `int sockets = (int) socketAffix != null ? socketAffix.level() : 0;`